example/capsnet/capsulenet.py

```diff
@@ -14,24 +14,31 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-import mxnet as mx
-import numpy as np
+"""
+Generate MXNet implementation of CapsNet
```

Comment:
Capsule Net
Can we add reference of the paper in comment?

Reply:
Copy that, it is good to help programmers to understand it.